modules/subscription/README.md

```diff
@@ -1 +1,168 @@
-# Subscription
\ No newline at end of file
+# Subscription Module
+This document introduces the **Subscription** module of the Sentinel Hub. Note that all examples use the official sentinel rpc server (https://rpc.sentinel.co:443). Obviously, any other rpc server can be used. Note further that all example transactions are signed with a local key named *myKeyringWallet*. Replace this with your own key name. Keys can be managed using the *keys* module. At the time of writing, the chain-id *sentinelhub-2* is used. This may change in the future.
+
+		
+# Queries
+The **Queries** submodule provides the logic to request information from subscriptions and quotas. The following messages are available:
+- [Subscription](#subscription)
+- [Subscriptions](#subscriptions)
+- [Quota](#quota)
+- [Quotas](#quotas)
+
+
+## Subscription
+
+**Description**
+
+Query a single subscription.
+
+**Usage**	
+
+	sentinelcli query subscription <SUBSCRIPTION_ID> [FLAGS]
```

Comment:
sentinelcli should not be here because modules directory should only have sentinelhub commands.

May be you can improve the guide for sentinelcli [here](https://github.com/sentinel-official/docs/blob/master/guides/clients/cli/COMMANDS.md)

Reply:
Sure. Do you want a dedicated file for the documentation or everything in the COMMAND.md? Putting everything in one file might get confusing. I could write the docs for the other sentinelcli modules, too, if you want.

Reply:
Yes, dedicated file for each module is good. Filename format: COMMANDS_<MODULE_NAME>.md

Or create a folder called commands and <MODULE_NAME>.md